packages/blade/src/components/Modal/Modal.native.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Text } from '~components/Typography';
+
+type ModalProps = {
+  children: React.ReactNode;
+  isOpen: boolean;
+  onDismiss: () => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  initialFocusRef?: React.MutableRefObject<any>;
+  size?: 'small' | 'medium' | 'large';
+  accessibilityLabel?: string;
+};
+
+const Modal = (): React.ReactElement => {
+  return (
+    <Text>
+      Modal Component is not available for Native mobile apps and we should use BottomSheet
```

Comment:
Should we throw an error instead?

Reply:
We had a discussion on this with Abinash, Kamlesh & Anurag where we concluded on throwing a warning since an error could be disruptive to end user in case a developer forgets to check their flow on mWeb and ships to prod